docs/resources/cluster.md

```diff
@@ -1,100 +1,334 @@
+  password      = resource.random_password.password.result
+  pg_config {
+    name  = "application_name"
+    value = "created through terraform"
+  }
+
+  pg_config {
+    name  = "array_nulls"
+    value = "off"
+  }
+
+  storage {
+    volume_type       = "gp3"
+    volume_properties = "gp3"
+    size              = "4 Gi"
+  }
+
+  pg_type            = "epas"
+  pg_version         = "14"
+  private_networking = false
+  cloud_provider     = "aws"
+  region             = "us-east-1"
+}
+
+output "password" {
+  sensitive = true
+  value     = resource.biganimal_cluster.eha_cluster.password
+}
+
+output "connection_uri" {
+  value = resource.biganimal_cluster.eha_cluster.connection_uri
+}
+```
 
 <!-- schema generated by tfplugindocs -->
 ## Schema
 
 ### Required
 
-- `cloud_provider` (String) Cloud Provider
-- `cluster_architecture` (Block List, Min: 1) Cluster Architecture (see [below for nested schema](#nestedblock--cluster_architecture))
+- `cloud_provider` (String) Cloud Provider. e.g. "aws" or "azure".
+- `cluster_architecture` (Block List, Min: 1) Cluster Architecture. See https://www.enterprisedb.com/docs/biganimal/latest/overview/02_high_availability/ for details. (see [below for nested schema](#nestedblock--cluster_architecture))
 - `cluster_name` (String) Name of the cluster.
-- `instance_type` (String) Instance Type
-- `password` (String, Sensitive) Password
-- `pg_type` (String) Postgres type
-- `pg_version` (String) Postgres Version
-- `region` (String) Region
-- `storage` (Block List, Min: 1) Storage (see [below for nested schema](#nestedblock--storage))
+- `instance_type` (String) Instance Type. e.g. "azure:Standard_D2s_v3", "aws:c5.large".
+- `password` (String, Sensitive) Password for the user edb_admin. It must be 12 characters or more.
+- `pg_type` (String) Postgres type. e.g. "epas", "pgextended" or "postgres".
+- `pg_version` (String) Postgres version. See https://www.enterprisedb.com/docs/biganimal/latest/overview/05_database_version_policy/#supported-postgres-types-and-versions for supported Postgres types and versions.
+- `region` (String) Region to deploy the cluster. See https://www.enterprisedb.com/docs/biganimal/latest/overview/03a_region_support/ for supported regions.
+- `storage` (Block List, Min: 1) Storage. (see [below for nested schema](#nestedblock--storage))
 
 ### Optional
 
-- `allowed_ip_ranges` (Block List) Allowed IP ranges (see [below for nested schema](#nestedblock--allowed_ip_ranges))
-- `backup_retention_period` (String) Backup Retention Period.
-- `pg_config` (Block List) Instance Type (see [below for nested schema](#nestedblock--pg_config))
-- `private_networking` (Boolean) Is private networking enabled
-- `read_only_connections` (Boolean) Is read only connections enabled
+- `allowed_ip_ranges` (Block List) Allowed IP ranges. (see [below for nested schema](#nestedblock--allowed_ip_ranges))
+- `backup_retention_period` (String) Backup Retention Period. e.g. "7d", "2w" or "3m".
```

Comment:
```suggestion
- `backup_retention_period` (String) Backup Retention Period. For example, "7d", "2w", or "3m".
```

Reply:
e.g. -> For example, fixed at https://github.com/EnterpriseDB/terraform-provider-biganimal/commit/4de4c956dd0831008781a5f59b9011542e378c5a
Serial comma, fixed at https://github.com/EnterpriseDB/terraform-provider-biganimal/commit/a7e9930ede32c6adc1fc0810c56a60d8391f3fb3